### PR TITLE
Update font-awesome icons

### PR DESCRIPTION
**Pull request by michplunkett**
## Description of Changes
This updates font-awesome to version 6 and add new twitter/instagram icon.

## Screenshots (if appropriate)
Screenshot 2023-12-11 at 9 48 57 PM

Screenshot 2023-12-11 at 9 49 11 PM

Screenshot 2023-12-11 at 9 49 19 PM

Screenshot 2023-12-11 at 9 49 23 PM

## Tests and linting
 - [x] This branch is up-to-date with the `develop` branch.
 - [x] `pytest` passes on my local development environment.
 - [x] `pre-commit` passes on my local development environment.

### CODE REVIEW

File: OpenOversight/app/templates/label_data.html [hunk 1]
```diff
@@ -100,52 +100,53 @@ <h2 class="horizontal-padding font-weight-300">How you can help</h2>
               <p class="lead margin-less">Sort Images</p>
               <div class="font-weight-300">Sort submitted images into those with and without officers.</div>
             </div>
-          <div class="col-lg-4 horizontal-padding""> <span class="fa fa-users fa-2x"></span>
```
Comment (michplunkett) [inline OpenOversight/app/templates/label_data.html:103]: Addressed the double `""` at the end of the `div` above this message, so there was a lot of formatting done around it.

File: OpenOversight/app/templates/label_data.html [hunk 2]
```diff
@@ -100,52 +100,53 @@ <h2 class="horizontal-padding font-weight-300">How you can help</h2>
...
-          <p class="lead margin-less">Identify Officers</p>
-          <div class="font-weight-300">Have a photo containing uniformed officers?</div>
-        </div>
-        <div class="col-lg-4 horizontal-padding"">
```
Comment (michplunkett) [inline OpenOversight/app/templates/label_data.html:107]: Addressed the double `""` at the end of the `div` above this message, so there was a lot of formatting done around it.

**Review by sea-kelp — CHANGES_REQUESTED** [review]
Can we use the regular or light variants of fontawesome instead of solid? The pencil-square and trash icons especially seem a bit dark on the site.

**label_data.html**
1. These three icons should use `fa-solid` instead of `fa` here
2. `fa-picture-o` doesn't seem to exist in the new version
https://github.com/lucyparsons/OpenOversight/blob/037debbdc0e4c36f8ef33e7d070e51465160d6a9/OpenOversight/app/templates/label_data.html#L99-L108

**officer_assignment_history.html**
This style should use `fa-solid` instead of `fa`
https://github.com/lucyparsons/OpenOversight/blob/037debbdc0e4c36f8ef33e7d070e51465160d6a9/OpenOversight/app/templates/partials/officer_assignment_history.html#L47

**Comment by michplunkett** [comment]
I gotchu, @sea-kelp! I'll make the changes sometime within the next two-three days.

File: OpenOversight/app/templates/label_data.html [hunk 3]
```diff
@@ -96,56 +96,57 @@ <h2>
           <h2 class="horizontal-padding font-weight-300">How you can help</h2>
           <div class="row pb-50">
             <div class="col-lg-4 horizontal-padding">
-              <span class="fa fa-picture-o fa-2x"></span>
+              <span class="fa-solid fa-image fa-2x"></span>
```
Comment (michplunkett) [inline OpenOversight/app/templates/label_data.html:99]: Screenshot 2023-12-18 at 9 41 55 PM

Here go, @sea-kelp!

File: OpenOversight/app/templates/partials/officer_assignment_history.html [hunk 4]
```diff
@@ -44,7 +44,7 @@ <h3>Assignment History</h3>
           {% if is_admin_or_coordinator %}
             <a href="{{ url_for('main.edit_assignment', officer_id=officer.id, assignment_id=assignment.id) }}">
               <span class="sr-only">Edit</span>
-              <i class="fa fa-pencil-square-o" aria-hidden="true"></i>
+              <i class="fa-solid fa-pencil-square" aria-hidden="true"></i>
```
Comment (michplunkett) [inline OpenOversight/app/templates/partials/officer_assignment_history.html:47]: We're good with the `fa-solid` here, @sea-kelp.

**Review by sea-kelp — APPROVED** [review]
Looks great! Thank you for making those changes

**Pull request merged**